docs/sources/getting-started/getting-started-prometheus.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Getting started with Grafana and Prometheus"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started"]
+type = "docs"
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+[menu.docs]
+identifier = "getting_started-grafana-prometheus"
+parent = "guides"
+weight = 300
++++
+
+# Getting Started with Grafana and Prometheus
+
+Prometheus is an open-source systems monitoring and alerting toolkit for which Grafana includes built-in support. This topic walks you through the steps to create a series of dashboards in Grafana displaying system metrics of a host monitored by Prometheus.
+
+You can also configure your Grafana instance in the cloud to display system metrics. For more information, refer to the [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) documentation.
+
+## Step 1: Install Grafana and build your first dashboard
+
+Use the instructions in [Getting started with Grafana]({{< relref "getting-started.md" >}}) to:
+- Install Grafana
+- Login to Grafana
+- Create your first dashboard
+
+## Step 2: Download Prometheus and node exporter
+
+Prometheus, like Grafana, can be installed on many different operating systems. Refer to the [Prometheus download page](https://prometheus.io/download/) which lists all stable versions of Prometheus components. Download the following components:
```

Comment:
Perhaps mention somewhere that Node Exporter doesn't come with a binary for MS Windows (and doesn't really work there – https://github.com/prometheus-community/windows_exporter is what to use there, but it might be too confusing to mention that in this tutorial).

Reply:
I will add this as a note. Thank you for sharing the workaround for Windows users.

Reply:
Added link to windows_exporter